### PR TITLE
go-github: move event types into event_types.go

### PR DESCRIPTION
**Pull request by gmlewis**
Change-Id: I9df89a4fc8d2213e5caf17599f752167386b98f3

### CODE REVIEW

File: github/event_types.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	Email *string `json:"email,omitempty"`
+}
+
+//PullRequestEvent represents the payload delivered by PullRequestEvent webhook
+type PullRequestEvent struct {
+	Action      *string      `json:"action,omitempty"`
+	Number      *int         `json:"number,omitempty"`
+	PullRequest *PullRequest `json:"pull_request,omitempty"`
+
+	// The following fields are shared by Webhook events and the Events API.
```
Comment (willnorris) [inline github/event_types.go]: actually, it's the opposite of this, right?  The following fields are *only* populated in Webhook events.

File: github/event_types.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
```
Comment (willnorris) [inline github/event_types.go]: maybe add a file-level comment that these event types are shared between the Events API and used as Webhook payloads.

File: github/event_types.go (hunk 1 above)
Comment (gmlewis) [inline github/event_types.go]: Yes indeed.  Sorry about that.
Done.

File: github/event_types.go (hunk 2 above)
Comment (gmlewis) [inline github/event_types.go]: Done.

**Pull request closed without merging**